demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -78,7 +78,7 @@
     INCIDENT_FIELD
 from TestSuite.pack import Pack
 from TestSuite.test_tools import ChangeCWD
-
+from pathlib import PosixPath
```

Comment:
```suggestion
from pathlib import Path
```

Reply:
It needs to specifically be PosixPath as that is the type that is returned from Git

Reply:
this is the type returned from git on MacOS/UNIX machines. To ease usage on other OSes, we better use `Path`. See more [here](https://docs.python.org/3/library/pathlib.html).